Configuration/Applications/python/ConfigBuilder.py

```diff
@@ -2217,6 +2217,16 @@ def prepare(self, doChecking = False):
 
 	self.pythonCfgCode += self.addCustomiseCmdLine()
 
+        # Temporary hack to put the early delete customization after
+        # everything else
+        #
+        # FIXME: remove when no longer needed
+        if "RECO" in self.stepMap or "RAW2RECO" in self.stepMap:
+                self.pythonCfgCode += "\n# Add early deletion of temporary data products to reduce peak memory need\n"
+                self.pythonCfgCode += "from RecoTracker.Configuration.customiseEarlyDeleteForSeeding import customiseEarlyDeleteForSeeding\n"
+                self.pythonCfgCode += "process = customiseEarlyDeleteForSeeding(process)\n"
```

Comment:
I'd rather we have more generic function calls (names) in ConfigBuilder.py so that it is not edited every time we decide to put something else in the early delete.

Configuration/StandardSequences/python/earlyDeleteSettings_cff.py may be a good name for that.